src/main/java/core/basesyntax/FigureTypeName.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum FigureTypeName {
```

Comment:
don't need it

Reply:
vs enaum is more visabillity, like vs colors